runtime/doc/windows.txt

```diff
@@ -392,6 +392,11 @@ CTRL-W CTRL-O					*CTRL-W_CTRL-O* *:on* *:only*
 		given, then they become hidden.  But modified buffers are
 		never abandoned, so changes cannot get lost.
 
+							*:fc* *:fclose*
+:[count]fc[lose][!]
+		Close [count]th floating window by zindex order. '!' to close
+		all floating windows.
```

Comment:
"order" seems to be the key word. It's not obvious that the zindex value itself is only relevant _relative_ to other zindexes. So if there are zindexes 99, 999, 9999, 

- `:2fclose` closes two windows (not obvious _which_ windows).
- `:fclose` closes all floats if cursor is _not_ in a float
- `:fclose` closes the highest-zindex float if cursor is in a float